docs/src/tut/wfs/index.md

```diff
@@ -0,0 +1,879 @@
+In an actual deployment, the seller would need to search for a contract name and its associated contract information in some repository. 
+
+Follow these directions to have the seller deploy the contract and return the contract information:
+
+1. In `index.mjs`, find the following code:
+
+    ``` js
+    load: /examples/docs-wisdom-7/index.mjs
+    range: 23-24
+    ```
+
+1. Replace it with the following:
+
+    ``` js
+    load: /examples/docs-wisdom-8/index.mjs
+    range: 23-28
+    ```
+
+    You will add a `wisdom` property to the `sellerInteract` later in the tutorial.
+
+1. In `index.rsh`, find the following code:
+
+    ``` reach
+    load: /examples/docs-wisdom-8/index.rsh
+    range: 4-5
+    ```
+
+1. Add the following `price` and `reportReady` Reach code:
+
+    ``` reach
+    load: /examples/docs-wisdom-8/index.rsh
+    range: 4-8
+    ```
+    
+    * Line 1: `sellerInteract` is a user-defined Reach object.
+    * Line 2: The spread syntax `...` adds all `commonInteract` properties (none yet) to the object.
+    * Line 3: `price ` is a `UInt`, a Reach-defined unsigned integer. 
+    * Line 4: `reportReady` is a function that takes a `UInt` as an argument and returns nothing.
+
+1. In `index.rsh`, add the following code between `{!rsh} init` and `{!rsh} exit`:
+
+    ``` reach
+    load: /examples/docs-wisdom-8/index.rsh
+    range: 16-23
+    ```
+
+    * Line 1: `init` initializes the DApp, and transitions to a step.
+    * Line 3: `S.only()` transitions to a local step in which seller gets `price`.
+    * Line 4: `S.publish()` transitions to a consensus step.
+    * Line 5: `S.interact` transitions to a local step in which seller passes `price` to frontend.
+    * Line 6: `commit()` transitions to a step.
+    * Line 8: `exit()` halts the contract forever.
+
+    The next section explains these functions in more detail.
+	For now, know that Reach programs (the `index.rsh` portion of your Reach DApp) are organized into four modes, and that `{!rsh} init`, `{!rsh} only`, `{!rsh} publish`, `{!rsh} commit`, and `{!rsh} exit` cause mode transitions.
+
+1. Run your DApp as the seller.
+   Output should resemble the following:
+
+    ```
+    Your role is seller.
+    The consensus network is ALGO.
+    Your balance is 1000 ALGO.
+    Your wisdom is for sale at 5 ALGO.
+    Contract info: {"type":"BigNumber","hex":"0xa6"}
+    Your balance is 999.996 ALGO.
+    ```
+
+    The seller creates the contract, retrieves the contract information, and makes it available to the buyer who will (shortly) use the information to attach to the contract.
+	Deploying the contract cost the seller a fee.
+
+The interact objects introduced in this section facilitate communication between the frontend (e.g. `index.mjs`) and backend (e.g. `index.main.mjs`) of Reach applications, (remembering that `index.rsh` is the pre-compiled version of `index.main.mjs`).
+
+# {#wfs-16} Attach to the contract (buyer)
```

Comment:
This is a good section to add an intuitive explanation about interactive command-lines and how we accomplish that with the `ask` object.